### DISCUSSION TITLE
Reload command

### DISCUSSION THREAD

**Discussion by groundbreakingmc** [PaperMC/Paper#11146] [Ideas]
On /reload command, call event like CoreReloadEvent, and than plugin can listen this event. Like:

```Java
@EventHandler
public void relodEvent(CoreReloadEvent e)
{
    if (e.isCanceled) return;

    Main.getInstance().relo-dConfig();
}
```

**Comment by lynxplay** [comment]
https://jd.papermc.io/paper/1.21/io/papermc/paper/event/server/ServerResourcesReloadedEvent.html is already called when minecraft:reload is executed.

An event for spigots broken reload is useless, your plugin will already be disabled.
